game_eggs/steamcmd_servers/ark_survival_evolved/egg-ark--survival-evolved.json

```diff
@@ -4,13 +4,13 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2021-07-27T14:22:41+03:00",
+    "exported_at": "2021-11-26T13:48:52+01:00",
     "name": "Ark: Survival Evolved",
     "author": "dev@shepper.fr",
     "description": "As a man or woman stranded, naked, freezing, and starving on the unforgiving shores of a mysterious island called ARK, use your skill and cunning to kill or tame and ride the plethora of leviathan dinosaurs and other primeval creatures roaming the land. Hunt, harvest resources, craft items, grow crops, research technologies, and build shelters to withstand the elements and store valuables, all while teaming up with (or preying upon) hundreds of other players to survive, dominate... and escape! \u2014 Gamepedia: ARK",
     "features": null,
     "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
+        "ghcr.io\/parkervcp\/games:source"
```

Comment:
This is a breaking change. There's no RCON cli in this image for the egg to work so it definitely couldn't have passed your startup test

Reply:
this fix this: https://github.com/parkervcp/yolks/pull/27